android/src/test/java/com/revenuecat/purchases/hybridcommon/CommonKtTests.kt

```diff
@@ -65,4 +66,113 @@ internal class CommonKtTests {
             setProxyURLString("this is not a url")
         }
     }
+
+    @Test
+    fun `calling canMakePayments correctly passes through to Purchases`() {
+        configure(
+                context = mockContext,
+                apiKey = "api_key",
+                appUserID = "appUserID",
+                observerMode = true,
+                platformInfo = PlatformInfo("flavor", "version")
+        )
+
+        every { Purchases.canMakePayments(mockContext, any(), any()) } just runs
```

Comment:
this will always return the same value, right? and we're not really checking that the value returned by `Purchases.canMakePayments` is the same one returned by `CommonKt.canMakePayments`. 
Perhaps we could have a random bool variable here, have `Purchases.canMakePayments` return the random bool, and check that the same value is received in `onResult.onReceived` (instead of matching to `any()`